### PR TITLE
[improvement](load) print more load log

### PR DESCRIPTION
**Pull request by dataroaring**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/exec/tablet_sink.cpp [hunk 1]
```diff
@@ -346,6 +331,23 @@ Status NodeChannel::add_row(BlockRow& block_row, int64_t tablet_id) {
     return Status::OK();
 }
 
+void NodeChannel::_sleep_if_memory_exceed() {
+    size_t begin_us = _mem_exceeded_block_ns;
+    while (!_cancelled && _pending_batches_bytes > _max_pending_batches_bytes) {
+        SCOPED_ATOMIC_TIMER(&_mem_exceeded_block_ns);
+        SleepFor(MonoDelta::FromMilliseconds(10));
+        if (_mem_exceeded_block_ns - begin_us > 5000000) {
```
Comment (cambyzju) [inline be/src/exec/tablet_sink.cpp:339]: a little confused: ns - us

**Comment by jacktengg** [comment]
LGTM

**Comment by cambyzju** [comment]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**